### ISSUE TITLE
Survey for Medical Staff (Google Forms)

### ISSUE THREAD

**Issue by DavidCTaylorPhD** [Charlotte-MEDI/Faceshields#19]
[Feedback Survey for Medical Staff that has no google sign-in](https://docs.google.com/forms/d/1TSu7uLs14O1G6E4whaSFB3JDFm9vUgBeI-um7PsmGt4/edit?usp=sharing)
[Feedback Survey for Medical Staff that has google sign-in](https://docs.google.com/forms/d/1f1yQcWOaGO_e_ZMt9W13Po1ughF1EaBVwzVy3lqsi9k/edit?usp=sharing)

The difference between these two forms is due to the "google sign-in" survey having the option for individuals to add images or videos, which is a mandatory Google Form feature. The "no google sign-in" survey does not have the option of image and video uploading. Google Forms requirement login when uploading files. Please let me know what you think. I can make any changes necessary. You all have full editing ability through the above links.

**Comment by danjstone** [comment]
In my own testing RC2 is not only more comfortable but also does not fog up as much

May want to add a question on fogging

**Comment by DavidCTaylorPhD** [comment]
Dan @danjstone

Great point! I added a fogging question to both surveys. Let me know what you think. I appreciate the help.

**Comment by JeremyProffitt** [comment]
@dtay1123 is there a way to see the results of the survey?

**Comment by DavidCTaylorPhD** [comment]
@JeremyProffitt yes you can. Once someone completes the survey it will populate with charts and data under the “Responses” tab near the top of page. Furthermore, we can export the data to a google sheets aka a spreadsheet for further statistical analysis automatically.  Thanks.

**Comment by DavidCTaylorPhD** [comment]
We have our first response to the survey.  [Google sign-n face shield survey responses](https://docs.google.com/forms/d/1f1yQcWOaGO_e_ZMt9W13Po1ughF1EaBVwzVy3lqsi9k/edit#responses)
